intermediate_notebooks/E2E/mortgage/mortgage_e2e.ipynb

```diff
@@ -90,6 +92,49 @@
     "client"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Use `pynvml` to determine the GPU memory and decide `start_year`, `end_year` and `part_count`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pynvml.nvmlInit()\n",
```

Comment:
Realized that this only works for multiple GPU's in the node running the notebook and not related to the GPU's used in the cluster.
TODO: Make this a method and wrap with a `client.submit` call